js/router.js

```diff
@@ -1,22 +1,23 @@
 import $ from 'jquery';
 import { ipcRenderer } from 'electron';
 import { Router } from 'backbone';
+import app from './app';
```

Comment:
Just reorganized some imports here. In general, I think it's best to keep views with views, utils with utils, models with models.

In a lot of files I'm seeing that we just dump new imports at the end as they are needed.